src/signals/incident-management/containers/IncidentDetail/components/StatusForm/styled.tsx

```diff
@@ -2,85 +2,114 @@
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
 import styled from 'styled-components'
 import {
+  Alert,
   Heading,
-  themeSpacing,
+  Label,
   Row,
-  Column,
   themeColor,
-  Label,
+  themeSpacing,
 } from '@amsterdam/asc-ui'
 
 import Button from 'components/Button'
 
+export const AddNoteWrapper = styled.div`
+  label {
+    display: none;
```

Comment:
Removing the label from the visible parts of the GUI will likely lead to an orphaned label; can you confirm that clicking the textarea label will still focus the textarea?

Reply:
I have refactored the label making sure that clicking the label will focus the textarea.